src/plugins/intel_npu/tests/functional/behavior/npuw/models/online_partitioning.hpp

```diff
@@ -0,0 +1,209 @@
+        convert[0] = std::make_shared<ov::op::v0::Convert>(model_c[15], ov::element::f16);
+        convert[1] = std::make_shared<ov::op::v0::Convert>(convert[0], ov::element::i32);
+        convert[2] = std::make_shared<ov::op::v0::Convert>(model_c[12], ov::element::i32);
+
+        for (const auto& c : convert) {
+            m_nodes.push_back(c);
+            set_name(c);
+        }
+
+        // Ops
+        std::vector<std::shared_ptr<ov::Node>> op(16, nullptr);
+        op[0] = std::make_shared<ov::op::v0::MatMul>(input, convert[1], false, true);
+        op[1] = std::make_shared<ov::op::v1::Reshape>(op[0], model_c[16], false);
+        op[2] = std::make_shared<ov::op::v1::Transpose>(op[1], model_c[0]);
+        op[3] = std::make_shared<ov::op::v0::ShapeOf>(op[2]);
+        op[4] = std::make_shared<ov::op::v1::Gather>(op[3], model_c[1], model_c[2]);
+        op[5] = std::make_shared<ov::op::v1::Divide>(op[4], model_c[3], true);
+        op[6] = std::make_shared<ov::op::v0::Floor>(op[5]);
+        op[7] = std::make_shared<ov::op::v3::ScatterUpdate>(model_c[5], model_c[6], op[6], model_c[7]);
+        op[8] = std::make_shared<ov::op::v1::StridedSlice>(op[2],
+                                                            model_c[8],
+                                                            op[7],
+                                                            model_c[9],
+                                                            std::vector<int64_t>{1, 1, 1, 1},
+                                                            std::vector<int64_t>{1, 1, 1, 1});
+        op[9] = std::make_shared<ov::op::v1::StridedSlice>(op[2],
+                                                            op[7],
+                                                            model_c[10],
+                                                            model_c[11],
+                                                            std::vector<int64_t>{1, 1, 1, 1},
+                                                            std::vector<int64_t>{1, 1, 1, 1});
+        op[10] = std::make_shared<ov::op::v1::Multiply>(op[9], convert[2]);
+        op[11] = std::make_shared<ov::op::v0::Concat>(std::vector<std::shared_ptr<ov::Node>>{op[10], op[8]}, -1);
+        op[12] = std::make_shared<ov::op::v1::Multiply>(model_c[13], op[11]);
+        op[13] = std::make_shared<ov::op::v1::Multiply>(model_c[14], op[2]);
+        op[14] = std::make_shared<ov::op::v1::Add>(op[13], op[12]);
+        op[15] = std::make_shared<ov::op::v1::Reshape>(op[14], model_c[17], false);
+
+        for (const auto& o : op) {
+            m_nodes.push_back(o);
+            set_name(o);
+        }
+
+        return op[15];
+    }
+
+private:
+    void set_name(const std::shared_ptr<ov::Node>& node) {
+        node->set_friendly_name("node_" + std::to_string(m_name_idx++));
+    }
+
+    std::vector<std::shared_ptr<ov::Node>> m_nodes;
+    size_t m_name_idx;
+};
+
+
+} // anonymous namespace
```

Comment:
Missing line at the end of file.

Reply:
Fixed!

Reply:
This file was also removed! Will be pushed in next PR!